ui/app/mainui/AppMain.qml

```diff
@@ -67,11 +67,11 @@ Item {
             keycardPopup.active = false
         }
 
-        function onMailserverNotWorking() {
+        onMailserverNotWorking: {
```

Comment:
Isn't the `function` syntax the only accepted syntax in 5.15? iirc, it shows deprecation warnings to not use a function

Reply:
Yeah it is but for some reason it doesn't work :/ Just keep the "function" signature, insert some `console.warn()` message and see it never called